app.fotema.Fotema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "app.fotema.Fotema",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.llvm18"
+  ],
+  "command": "fotema",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures:ro",
+    "--env=RUST_BACKTRACE=1"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm18/bin"
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/lib/cmake",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "add-extensions": {
+    "app.fotema.Fotema.HEIC": {
+      "directory": "lib/libheif-heic",
+      "add-ld-path": "lib",
+      "bundle": true,
+      "autodelete": true
+    }
+  },
+  "modules": [
+    "modules/libde265.json",
+    "modules/libheif.json",
+    "modules/glycin-loaders.json",
+    "modules/ffmpeg.json",
```

Comment:
Please add these to the main manifest here.

Reply:
Done.

Reply:
You can but keeping them in the main manifest is better.

Reply:
you shoudln't. it makes it hard to maintain. and when a maintainer goes MIA it team member like @bbhtt and I that pick up the pieces.